### DISCUSSION TITLE
How to update to latest version?

### DISCUSSION THREAD

**Discussion by iamdineshkumar** [coollabsio/coolify#455] [Q&A] [answered]
I have installed a version in ubuntu based on script provided but its not upgrading to latest version even after trying to run same script again. I even tried to add -a on command for auto upgrade but its not workout also. Guide me on that.

**Comment by andrasbacsai** [comment]
I think this has already been discussed on Discord. LMK if not. 🙂

**Reply by iamdineshkumar** [reply]
@andrasbacsai Yes...

**Reply by iamdineshkumar** [reply]
Just shared answer below for easy reference of others

**Comment by iamdineshkumar** [comment, marked as answer]
I am sharing below answers for others who is checking this thread. Update can be done in two ways.
1. Easy way (Without Terminal), In dashboard menu, update button will be displayed after login into coolify by admin. Click on it and sit back and relax in few seconds things will be done.
![image](https://user-images.githubusercontent.com/14805534/174593373-f85bd4ad-ced8-4844-87be-23ecfa7fa966.png)

2. Nothing complex jut need to use terminal and use same command used for install coolify add -f flag if you need to automate it
